ElementX/Sources/Services/Timeline/TimelineItemProxy.swift

```diff
@@ -142,3 +129,44 @@ extension TimelineItemContentKind {
         }
     }
 }
+
+struct TimelineItemDebugInfo: Identifiable, CustomStringConvertible {
+    let id = UUID()
+    let model: String
+    let originalJson: String?
+    let latestEditJson: String?
+    
+    init(model: String, originalJson: String?, latestEditJson: String?) {
```

Comment:
Just doing a few of these, but I would say that in Swift, we should capitalise this ala `JSONEncoder` `JSONSerialization` etc etc.
```suggestion
    let originalJSON: String?
    let latestEditJSON: String?
    
    init(model: String, originalJSON: String?, latestEditJSON: String?) {
```